app/src/main/java/com/android/voyageur/model/user/UserViewModel.kt

```diff
@@ -88,21 +102,31 @@ open class UserViewModel(
         if (firebaseUser != null) {
           // User is signed in, start listening to user data
           loadUser(firebaseUser.uid, firebaseUser)
-          // Fetch sent friend requests
+          // Listen to friend requests (incoming)
+          listenToFriendRequests()
+          // Listen to friend requests (sent)
           getSentFriendRequests()
         } else {
-          // User is signed out, clear user data and remove listeners
+          // User is signed out, clear data and remove listeners
```

Comment:
Good catch to check the sign out behaviour

Reply:
Thank you!